### PR TITLE
LST-8 Setup telegram bot with grammy framework

### LINKED ISSUE #8: Setup Telegram Bot
Use this tutorial 
- https://www.digitalocean.com/community/tutorials/how-to-build-a-telegram-quotes-generator-bot-with-node-js-telegraf-jimp-and-pexels
- https://www.cyclic.sh/posts/how-to-build-a-telegram-bot/ (Grammy + TS)

### PR DESCRIPTION
**Pull request by alimsanzhar**
This PR sets up minimal telegram bot with grammy framework. Based on this [article](https://www.cyclic.sh/posts/how-to-build-a-telegram-bot/).

First of all we need to setup .env file to run telegram bot. We decided to store it locally as it stores sensitive data. Create locally **.env** file near **.env.example** file and paste all the stuff from the **.env.example** file to **.env** file and  replace *** to real token. Then run **yarn dev**.

Closes #8

### CODE REVIEW

File: packages/telegram-bot/nodemon.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "watch": ["src"],
   "ext": "ts,js,json",
   "ignore": ["coverage"],
-  "exec": "tsc && ts-node ./src/server.ts",
+  "exec": "tsc && env-cmd ts-node src/index.ts",
```
Comment (alimsanzhar) [inline packages/telegram-bot/nodemon.json:5]: **env-cmd** - allows to run configs from .env file

File: packages/telegram-bot/package.json [hunk 2]
```diff
@@ -2,18 +2,25 @@
...
-    "start": "yarn build && node dist/server.js",
+    "start": "yarn build && node .",
     "test": "yarn build && jest",
     "test:coverage": "yarn test -- --coverage --collectCoverageFrom=\"./src/**\"",
     "test:watch": "yarn build && jest --watch"
   },
   "devDependencies": {
+    "@types/debug": "^4.1.7",
+    "@types/node": "^18.11.18",
+    "@types/node-fetch": "^2.6.2",
```
Comment (alimsanzhar) [inline packages/telegram-bot/package.json:17]: Included these dependencies to the project because grammy requires them

File: .eslintrc.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ module.exports = {
     'import/no-unresolved': 'error',
     'no-console': 'off',
     'import/prefer-default-export': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```
Comment (oizuldan) [inline .eslintrc.js]: do not turn off this rule, use the type `unknown` if you don't know the exact type

File: .gitignore [hunk 4]
```diff
@@ -72,7 +72,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
```
Comment (oizuldan) [inline .gitignore]: it is correct, we should not expose our environment files, especially production ones to outside world. I advise you to create a `.env.example` file where we will expose public envs (such as staging creds). Devs then will need to create their own version of `.env` file based on example file. By doing so, we will be sure to keep crucial tokens out of reach of regular users

File: packages/telegram-bot/nodemon.json (hunk 1 above)
Comment (oizuldan) [reply packages/telegram-bot/nodemon.json:5]: if I am not mistaken node itself can access env files using `process.env.NAME_OF_ENV`. Check @nurbekismagulov 's PR for a reference

```
const port = Number(process.env.PORT) || 5000;
```

File: packages/telegram-bot/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+const bot = new Bot(process.env.TELEGRAM_TOKEN || '');
+
+const introductionMessage = `Hello! I'm a Telegram bot.
+I'm powered by Cyclic, the next-generation serverless computing platform.
+
+<b>Commands</b>
+/yo - Be greeted by me
+/effect [text] - Show a keyboard to apply text effects to [text]`;
+
+const replyWithIntro = (ctx: any) =>
```
Comment (oizuldan) [inline packages/telegram-bot/src/index.ts]:
```suggestion
import { Bot, Context } from 'grammy';

const bot = new Bot(process.env.TELEGRAM_TOKEN || '');

const introductionMessage = `Hello! I'm a Telegram bot.
I'm powered by Cyclic, the next-generation serverless computing platform.

<b>Commands</b>
/yo - Be greeted by me
/effect [text] - Show a keyboard to apply text effects to [text]`;

const replyWithIntro = (ctx: Context) =>
```

**Review by oizuldan — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (alimsanzhar) [reply .gitignore]: Done. Decided to store sensitive credentials locally and created **.env.example** file to show how to work with **.env** file locally (detail info in PR description).

File: packages/telegram-bot/nodemon.json (hunk 1 above)
Comment (alimsanzhar) [reply packages/telegram-bot/nodemon.json:5]: As we already discussed in private to read data from env file we need this package. Another analogy for this package is dotenv. The main difference is that dotenv is called in code while env-cmd is called in script.

**Review by oizuldan — APPROVED** [review]

**Pull request merged**